### DISCUSSION TITLE
Alternative to cast `as Meta` in default export?

### DISCUSSION THREAD

**Discussion by bodadotsh** [storybookjs/storybook#14317] [Documentation]
In TypeScript, [the docs](https://storybook.js.org/docs/react/writing-stories/introduction#default-export) recommends to cast `as Meta` in default export

```ts
export default {
  title: 'components/Card',
  component: Card,
  foo: () => {}, // not warned here!
  decorators: [
    Story => (
      <Box p="xl4">
        <Story />
      </Box>
    ),
  ],
} as Meta;
```

Notice how the random `foo` property isn't warned?

But, if we do this instead

```ts
const StorybookMeta: Meta = {
  title: 'components/Card',
  component: Card,
  foo: () => {}, // Object literal may only specify known properties, and 'foo' does not exist in type 'Meta<Args>'.ts(2322)
  decorators: [
    Story => (
      <Box p="xl4">
        <Story />
      </Box>
    ),
  ],
};

export default StorybookMeta;
```

We will get TS errors.

Thoughts?

**Comment by IanVS** [comment]
So far I'm using the approach in https://github.com/storybookjs/storybook/issues/13747, and it seems to work okay for my specific situation.
